tests/include/igl/cr_vector_curvature_correction.cpp

```diff
@@ -0,0 +1,84 @@
,-0.370240244846531,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.370240244846531,0.370240244846531,0,0,0,0,0,0,0,5.55111512312578e-17,0,-5.55111512312578e-17,0,0,0,0,0,2.09439510239319,-0.370240244846531,0.370240244846531,0,0,0,0,0,0,0,0,0,0,0,0,-0.370240244846531,0,0,-0.370240244846531,0,0,0,0.37024024484653,0,0,0,0,0,-0.37024024484653,0,0,0,0,0.370240244846531,0,0,-0.370240244846531,2.0943951023932,0,0,0.37024024484653,0,0,0,0,0,0.37024024484653,0,0,0,0,0,0,-0.370240244846531,-0.370240244846531,0,0,0,0,0,0,0,0,0.37024024484653,0,0,-0.37024024484653,0,0,0,0,-0.370240244846531,0.370240244846531,0,2.0943951023932,0,0,0,0,0,0,-0.37024024484653,0,0,-0.37024024484653,0,0,-5.55111512312578e-17,0.37024024484653,0,0,0,0,0,0,0.37024024484653,0,5.55111512312578e-17,0,0,0,0,0,0,0,0,0.37024024484653,0,0,0,0,2.0943951023932,0,0.37024024484653,0,0,0,0,0,0,0,0,0,-5.55111512312578e-17,0,0,0,-0.37024024484653,0,0,0,0.37024024484653,0,0,0,0,0,0,0,-5.55111512312578e-17,0,0,0,0,0,0.37024024484653,0,0,2.0943951023932,-0.37024024484653,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-0.37024024484653,-0.37024024484653,0,0,0.370240244846531,0,0,0,0,0,0.370240244846531,0,0,0,0,0,0,0,0.37024024484653,-0.37024024484653,2.0943951023932,0,0.370240244846531,0,0,0,0,0,-0.370240244846531,0,0,-0.37024024484653,5.55111512312578e-17,0,0,0,0,0,0,0,-5.55111512312578e-17,0.37024024484653,0,0,0,0,0,0,0,0.37024024484653,0,0,0,0,0,0,0,2.0943951023932,0,-0.37024024484653,0,0,0,0,0,0,0,0,0,0,0,0,-5.55111512312578e-17,0,-0.370240244846531,5.55111512312578e-17,0,-0.370240244846531,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.370240244846531,0,2.09439510239319,-0.370240244846531,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-0.37024024484653,0.370240244846531,0,-0.37024024484653,0,0,0,0,0.370240244846531,0,0,0,0,0,0,0,0,0,-0.37024024484653,-0.370240244846531,2.0943951023932,0.37024024484653,0,0,0,0,0.370240244846531,0,0,5.55111512312578e-17,0,0,-0.37024024484653,0,0,0,0,0,0.37024024484653,0,0,0,0,0,5.55111512312578e-17,0,0,0,0,0,-0.37024024484653,0,0,0,0,0,0.37024024484653,2.0943951023932,0,0,0,0,0,0,0,0,-5.55111512312578e-17,0.37024024484653,0,0,0,0,0,0,0,0,0,0.37024024484653,0,5.55111512312578e-17,0,0,0,0,0,0.37024024484653,0,0,0,0,0,0,0,0,2.0943951023932,0.37024024484653,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-0.37024024484653,0,0.37024024484653,0.370240244846531,-0.370240244846531,0,0,0,0,0,0,0,0,0,0,0,0,0,0.37024024484653,2.0943951023932,0.37024024484653,0.370240244846531,0.370240244846531,0,0,0,5.55111512312578e-17,0,0.37024024484653,0,0,0,0,0,0,0,0,-0.37024024484653,0,0,-5.55111512312578e-17,0,0,0,0,0,-0.37024024484653,0,0,0,0,0,0,0,0,0.37024024484653,2.0943951023932,0,0,0,0,0,0,0,0,0,5.55111512312578e-17,-0.370240244846531,0,0,0,0,-5.55111512312578e-17,-0.370240244846531,0,0,0,0,0,0,0,0,0,0,0,-0.370240244846531,0,0,0,0,0,0.370240244846531,0,2.09439510239319,0,0,0,0,0,0,0,0,0,0,0,0,-0.370240244846531,-5.55111512312578e-17,0,0.370240244846531,5.55111512312578e-17,0,0,0,0,0,0,0,0,0,0,0,0,0,0.370240244846531,0,0,0.370240244846531,0,0,2.09439510239319;
```

Comment:
FYI, if you need to enter binary data in C++ code, you can dump it with `std::cout << std::hexfloat` first (that would avoid conversion to base 10 and back).